cdap-ui/app/features/hydratorplusplus/controllers/create/leftpanel-ctrl.js

```diff
@@ -136,9 +136,9 @@ class HydratorPlusPlusLeftPanelCtrl {
       return connections;
     };
 
-    let isValidArtifact = (importArtifact) => {
-      return this.artifacts.filter( artifact => angular.equals(artifact, importArtifact)).length;
-    };
+    // let isValidArtifact = (importArtifact) => {
+    //   return this.artifacts.filter( artifact => angular.equals(artifact, importArtifact)).length;
+    // };
```

Comment:
So have commented it out for now. We still need to validate the json that the user is importing. Didn't want to add that to the same PR as it is getting too big. For now I am by-passing the validation during import.